frontend/src/metabase/redux/downloads.ts

```diff
@@ -206,20 +206,23 @@ const getDatasetParams = ({
       return {
         method: "GET",
         url: `/api/embed/dashboard/${token}/dashcard/${dashcardId}/card/${cardId}/${type}`,
-        params: Urls.getEncodedUrlSearchParams({ ...params, ...exportParams }),
+        params: new URLSearchParams({
+          parameters: JSON.stringify(params),
+          ..._.mapObject(exportParams, value => String(value)),
```

Comment:
`exportParams` needs to have values as strings if we want to pass it to `URLSearchParams`